src/tabs/motors.html

```diff
@@ -89,8 +89,12 @@
                                 <div class="number checkboxDshotBidir">
                                     <div style="float: left; height: 20px; margin-right: 14px; margin-left: 3px;">
                                         <input type="checkbox" id="dshotBidir" class="toggle" />
+	                                    <span class="freelabel" i18n="configurationDshotBidir"></span>
+                                    </div>
+                                    <div style="float: left; height: 20px; margin-right: 14px; margin-left: 3px;" class="checkboxDshotEdt">
+                                        <input type="checkbox" id="dshotEdt" class="toggle" />
+	                                    <span class="freelabel" id="dshotEdtLabel" i18n="configurationDshotEdt"></span>
```

Comment:
Ooof, these floats look painful 🫠 But probably better for style consistency. More of a note to self/team 🙈